content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
Should this be under the new `citadel-config` sub-folder? https://preliminary.istio.io/docs/tasks/security/citadel-config/
If this replaces Citadel, I think we could say this is "configuring Citadel to be disabled", so it belongs in the sub-folder.

Reply:
This feature is orthogonal to Citadel; Citadel can exist as is, e.g., signing workload certificates, etc.

Reply:
In that case I guess it belongs where you have it. That it's unrelated to Citadel deployment, however, is not clear from the text. You list its advantage as being light weight, compared to Citadel, but that's irrelevant if you need to deploy Citadel as well anyway. So this is only used for the webhooks, while Citadel continues to provide certificates for workloads? You should explain this a little more clearly in the overview.

Reply:
There are users that do not use Citadel but their own CA to sign workload certificates. For such users, they can use Chiron, instead of Citadel, to provision DNS certificates for their control plane components, which is more lightweight than Citadel.

Reply:
Added an explanation that this is to provision and manage DNS certificates for Istio control
plane components.

Reply:
This raises the architectural question about whether Citadel needs to support provisioning DNS certs at all. If Chiron is there, why not always use that and remove that functionality from Citadel?

Reply:
That is the plan. This feature is alpha, so the current default behavior remains unchanged. After this feature graduates as beta, it will become the default and the corresponding functionality will be removed from Citadel.